grails-app/conf/application.yml

```diff
@@ -570,7 +570,7 @@ openboxes:
 
     # Allow users to customize logo image url as well as label
     logo:
-        url: "/assets/openboxes_logo_40x40.jpg"
+        url: "${server.contextPath}/assets/openboxes_logo_40x40.jpg"
```

Comment:
And this does not affect the `displayLogo` tag? I mean, that this won't produce there `/openboxes/openboxes/assets/openboxes_logo_40x40.jpg`?

Reply:
@awalkowiak Good point. I'll test it out, but I'm sure it does do that.

Can you and @alannadolny review ticket #5041 and see if you can think of a better approach?

Reply:
Perhaps, we just need to make the change on the API side like Approach 3 suggests.

Reply:
It looks like it works fine within displayLogo too.  Therefore I'm going to merge as-is

![image](https://github.com/user-attachments/assets/0fc5d24d-032b-4507-a9a0-b12b356d9d6e)